tests/data/spki.pem

```diff
@@ -0,0 +1,14 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
Out of curiosity, how was this generated?

Reply:
Did this a few weeks ago, but if memory serves me correctly with: `openssl rsa -in private_key.pem -pubout -out public_key.pem`.